src/components/Subtitle/SubtitleStream.scss

```
@@ -1,7 +1,12 @@
 .subtitle-stream {
   flex: 1;
   min-height: 0;
-  padding: 12px 24px 16px;
+  // 12px horizontal matches `.conversation-display` in MainPanel.scss. The
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)